staging/vhost-device-sound/src/device.rs

```diff
@@ -1111,16 +1114,16 @@ mod tests {
         exit.unwrap().write(1).unwrap();
 
         let ret = backend.handle_event(CONTROL_QUEUE_IDX, EventSet::IN, &vrings, 0);
-        assert!(!ret.unwrap());
+        assert!(ret.is_ok());
 
         let ret = backend.handle_event(EVENT_QUEUE_IDX, EventSet::IN, &vrings, 0);
-        assert!(!ret.unwrap());
+        assert!(ret.is_ok());
 
         let ret = backend.handle_event(TX_QUEUE_IDX, EventSet::IN, &vrings, 0);
-        assert!(!ret.unwrap());
+        assert!(ret.is_ok());
 
         let ret = backend.handle_event(RX_QUEUE_IDX, EventSet::IN, &vrings, 0);
-        assert!(!ret.unwrap());
+        assert!(ret.is_ok());
```

Comment:
Clippy is complaining about these changes, see https://buildkite.com/rust-vmm/vhost-device-ci/builds/1962#018bc964-fa22-4499-a69c-b8bdd330cf27

It is suggesting to use just `ret.unwrap()`.

Reply:
fixed... need to update my test.sh script with staging/...